code/game/machinery/washing_machine.dm

```diff
@@ -244,8 +244,10 @@ GLOBAL_LIST_INIT(dye_registry, list(
 	worn_icon_state = initial(target_type.worn_icon_state)
 	inhand_x_dimension = initial(target_type.inhand_x_dimension)
 	inhand_y_dimension = initial(target_type.inhand_y_dimension)
-	name = initial(target_type.name)
-	desc = "[initial(target_type.desc)] The colors look a little dodgy."
+	//indicate that this is a dyed object rather than the real thing
+	name = "dyed [initial(name)]"
+	var/dyed_warning = gender == PLURAL ? "They have been dyed to resemble" : "It has been dyed to resemble a"
```

Comment:
```suggestion
	var/dyed_warning = "[p_they(TRUE)] [p_have()] been dyed to resemble"
```
you can use pronoun procs for this, instead of needing to account plural vs non-plural explicitly